com.unity.UnityHub.yaml

```diff
@@ -0,0 +1,84 @@
+app-id: com.unity.UnityHub
+branch: stable
+base: io.atom.electron.BaseApp
+base-version: '18.08'
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+command: start-unityhub
+separate-locales: false
+tags: [proprietary]
+finish-args:
+  - '--allow=multiarch'
+  - '--device=dri'
+  - '--filesystem=host'
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=pulseaudio'
+  - '--socket=x11'
+  - '--talk-name=org.freedesktop.Flatpak'
+  - '--talk-name=org.gnome.Software'
+add-extensions:
+  org.freedesktop.Platform.Compat.i386:
+    directory: lib/i386-linux-gnu
+    version: '18.08'
+  org.freedesktop.Platform.Compat.i386.Debug:
+    directory: lib/debug/lib/i386-linux-gnu
+    version: '18.08'
+modules:
+  - name: compat
+    buildsystem: simple
+    build-commands:
+      - 'mkdir -p /app/lib/i386-linux-gnu /app/lib/debug/lib/i386-linux-gnu'
+
+  - shared-modules/glu/glu-9.0.0.json
+
+  - name: unappimage
+    buildsystem: simple
+    build-commands:
+      - 'make -C squashfs-tools install INSTALL_DIR=/app/bin'
+    sources:
+      - type: git
+        url: https://github.com/kirbyfan64/unappimage
+        commit: e3f52ebca3893e943ac97b955f47da626405424d
+
+  - name: cpio
+    cleanup:
+      - '/libexec'
+      - '/share/info'
+      - '/share/man'
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/cpio/cpio-2.12.tar.gz
+        sha256: 08a35e92deb3c85d269a0059a27d4140a9667a6369459299d08c17f713a92e73
+
+  - name: unityhub
+    buildsystem: simple
+    build-commands:
+      - 'install -Dm 755 unityhub.sh /app/bin/start-unityhub'
+      - 'install -Dm 755 vscode.py /app/bin/vscode-editor'
+      - 'install -Dm 755 apply_extra.sh /app/bin/apply_extra'
+      - 'install -Dm 755 patch-resources.py /app/bin/patch-resources'
+      - 'install -D com.unity.UnityHub.appdata.xml /app/share/appdata/com.unity.UnityHub.appdata.xml'
```

Comment:
`-Dm644`, same goes for the lower two lines.

Also, the AppData should go into `/app/share/metainfo`.